index.html

```diff
@@ -327,14 +327,11 @@ <h2>About me</h2>
             required
           ></textarea>
 
```

Comment:
- Please check the requirements for the form on this project. You are supposed to check:
  - The name field accepts up to 30 chars;
  - The email field accepts only emails;
  - The message field accepts up to 500 chars;
  - All fields have the correct type.